src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype/InternalCpxSvDiscoverFromLongReadsSpark.java

```diff
@@ -0,0 +1,228 @@
+package org.broadinstitute.hellbender.tools.spark.sv.discovery.prototype;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMRecord;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.ArgumentCollection;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.StructuralVariationSparkProgramGroup;
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+import org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.spark.sv.StructuralVariationDiscoveryArgumentCollection;
+import org.broadinstitute.hellbender.tools.spark.sv.discovery.AlignedContig;
+import org.broadinstitute.hellbender.tools.spark.sv.discovery.AlignmentInterval;
+import org.broadinstitute.hellbender.tools.spark.sv.utils.FileUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.util.EnumMap;
+import java.util.HashSet;
+import java.util.Set;
+
+
+/**
+ * This tool takes a SAM file containing alignments of single-ended long read
+ * (be it long read sequencing, or contigs assembled from standard Illumina short reads),
+ * searches for reads with complicated split alignments indicating the presence of complex structural variations (cxSV),
```

Comment:
Since we're also calling simple SVs (as done in this PR), this description seems incomplete.